spec/controllers/manuals_controller_spec.rb

```diff
@@ -22,5 +22,9 @@
     it "does not publish the manual" do
       expect(ManualsController).not_to have_received(:publish)
     end
+
+    it "sets the authenticated user header" do
+      expect(/uid-\d+/).to match(GdsApi::GovukHeaders.headers[:x_govuk_authenticated_user])
```

Comment:
These are supposed to be the other way round, not sure if it works properly like this?